jupyterdrive/gdrive/driveutils.d.ts

```diff
@@ -0,0 +1,103 @@
+import iface = require('content-interface');
+export declare var FOLDER_MIME_TYPE: string;
+export declare var NOTEBOOK_MIMETYPE: string;
+export declare var MULTIPART_BOUNDARY: string;
+export declare enum FileType {
```

Comment:
I cannot of it will change all import to `import driveutils_module =  require("driveutils")`
`var driveutils = driveutils_module.driveutils` which ca raise cannot to var in ambient declaration.